test/e2e/recordings/should-deploy-without-throwing-an-error.json

```diff
@@ -2,7 +2,7 @@
     {
         "scope": "https://deploy-cli-dev.eu.auth0.com:443",
         "method": "GET",
-        "path": "/api/v2/rules?include_totals=true&page=0&per_page=100",
```

Comment:
These recordings should probably be reverted once the pagination functionality is added back in.